app/assets/javascripts/front/realization.js

```diff
@@ -0,0 +1,253 @@
+    this._scrollTo(this.target)
+
+  }
+
+  setupDesktopPlayer() {
+    let jwConfig = {
+      aspectratio: '16:10',
+      autostart: false,
+      cast: {
+        appid: '00000000'
+      },
+      controls: true,
+      displaydescription: true,
+      displaytitle: true,
+      flashplayer: '//ssl.p.jwpcdn.com/player/v/8.17.7/jwplayer.flash.swf',
+      height: 360,
+      key: 'tlbC+HgyRBgC8seKreQwmghcO2P9nQ4y+l4ZeKQEfzpgOSzuT9f5+N0odL4=',
+      mute: false,
+      ph: 3,
+      pid: '8SfP7aMx',
+      playbackRateControls: false,
+      playlist: '//cdn.jwplayer.com/v2/media/1oDJAjaD?recommendations_playlist_id=xFiJh22s',
+      preload: 'metadata',
+      repeat: false,
+      stagevideo: false,
+      stretching: 'uniform',
+      width: '100%'
+    }
+    jwplayer('desktopVideo').setup(jwConfig)
+
+    this.loadJWPlayer('desktopVideo')
+  }
+
+  setupMobilePlayer() {
+    let jwConfig = {
+      aspectratio: '9:16',
+      autostart: false,
+      cast: {
+        appid: '00000000'
+      },
+      controls: true,
+      displaydescription: false,
+      displaytitle: false,
+      flashplayer: '//ssl.p.jwpcdn.com/player/v/8.17.7/jwplayer.flash.swf',
+      height: 360,
+      key: 'G4TQ74H6dL+J2s9zjsY53y72/VJdEKk5UO4K8F3CL4mUDDFs7Wl6siOex5c=',
+      mute: false,
+      ph: 3,
+      pid: 'NaHiexhY',
+      playbackRateControls: false,
+      playlist: '//cdn.jwplayer.com/v2/media/8FivThod?recommendations_playlist_id=xFiJh22s',
+      preload: 'metadata',
+      repeat: false,
+      stagevideo: false,
+      stretching: 'uniform',
+      width: '100%'
+    }
+    jwplayer('mobileVideo').setup(jwConfig)
+
+    this.loadJWPlayer('mobileVideo')
+  }
+
+  loadJWPlayer(div) {
+    let player = jwplayer(div)
+    this.target = 'video'
+
+    player.on('pause', () => {
+      this.target = 'paused'
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+    })
+
+    player.on('complete', () => {
+      this.target = 'survey'
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+    })
+
+    this._scrollTo(this.target)
+  }
+
+  passFeel() {
+    let feelings = []
+    let feelFlag = false
+    for (let i = 0; i < this.surveyBtns.length; i++) {
+      let surveyBtn = this.surveyBtns[i]
+      if (surveyBtn.classList.contains('button--active')) {
+        feelFlag = true
+        feelings.push(surveyBtn.id)
+      }
+    }
+    if (feelFlag) {
+      let props = {
+        'feel': feelings
+      }
+      this.target = 'course'
+      this.container.dataset.screen = this.target
+      this._scrollTo(this.target)
+
+      this.courseFeelInput.value = JSON.stringify(props)
+    } else {
+      $(this.surveyMsg).show()
+    }
+  }
+
+  formSending(btn) {
+    btn.innerText = 'Loading...'
+  }
+
+  formSuccess() {
+    this.target = 'thanks'
+    this.container.dataset.screen = this.target
+  }
+
+  formError() {
+    console.log('Error in form submission...')
+  }
+
+  share() {
```

Comment:
Could you move the `share` and `shareFallback` to `util.js` so that we might be able to easily reuse it in other parts of the site in the future.